spec/test_app.rb

```diff
@@ -55,4 +55,18 @@ class TestApp < Sinatra::Base
     </dl>
     HTML
   end
+
+  get '/elements-with-ids' do
+    <<-HTML
+    <div id="div-1"></div>
```

Comment:
The problem here is that the `<div>` doesn't have any content, which makes it invisible for chrome. Just add any text inside the div and the spec should pass